release_notes.md

```diff
@@ -1,14 +1,115 @@
-Release Notes for OpenMDAO 2.10.0
+**********************************
+# Release Notes for OpenMDAO 3.0.0
+
+March 05, 2020
+
+OpenMDAO 3.0.0 is the first version of OpenMDAO to drop support for Python 2.7.  Going forward,
+we will support Python >= 3.6.  Functionally, 3.0.0 is identical to 2.10.1 with the exception
+that long-standing deprecations have been removed and will now result in errors.
+
+When upgrading to OpenMDAO 3.0.0, users should
+1. Get their models working with version 2.10.1
+2. Make necessary changes to remove any OpenMDAO-specific deprecation warnings
+3. Upgrade to OpenMDAO 3.0.0 and verify that their models are working as expected.
+
+Note: This is the last release of OpenMDAO 2.X. Updating to this release will be a
+critical step in preparing for the OpenMDAO 3.X releases
+
+
+## Backwards Incompatible API Changes:
+
+The following are deprecated functionality since 2.0 that has been officially removed as of version 3.0.0:
+
+- <POEM 004> AkimaSplineComp and BsplineComp have been **removed**, replaced with SplineComp.
+
+- <POEM 008> The user is now **required** to set a value for the solver option `solve_subsystems`
+    on Newton and Broyden solvers. Previously the default was `False`, but you will now get
+    a deprecation warning if this value is not explicitly set.
+
+- The class for the PETSc linear solver `PetscKSP` is renamed `PetscKrylov`.
+
+- The class for the Scipy linear solver `ScipyIterativeSolver` is renamed `ScipyKrylov`.
+
+- The `NonLinearRunOnce` solver is now `NonlinearRunOnce`.
+
+- The `FloatKrigingSurrogate` is now `KrigingSurrogate`.
+
+- Optimizer `ScipyOptimizer` is now `ScipyOptimizeDriver`.
+
+- API function and CLI command `view_model` are renamed `n2`.
+
+- Component `ExternalCode` is renamed `ExternalCodeComp`.
+
+- Component `KSComponent` is renamed `KSComp`.
+
+- Component `MetaModelStructured` is renamed `MetaModelStructuredComp`.
+
+- Component `MetaModel` is renamed `MetaModelUnStructuredComp`.
+
+- Component `MultiFiMetaModel` is renamed `MultiFiMetaModelUnStructuredComp`.
+
+- Case attribute `iteration_coordinate` is renamed `name`.
+
+- Solver option `err_on_maxiter` is renamed `err_on_non_converge`.
+
+- Solver attribute `preconditioner` is renamed `precon`.
+
+- Nonlinear solver option `reraise_child_analysiserror` now defaults to False.
+
+- BroydenSolver and NewtonSolver linesearch now defaults to BoundsEnforceLS.
+
+- OptionsDictionary option argument `_type` is renamed `types`.
+
+- Component attribute `distributed` has been changed to an option.
+
+- The attribute `root` of Problem is renamed `model`.
+
+- Group attribute `nl_solver` is renamed `nonlinear_solver`.
+
+- Group attribute `ln_solver` is renamed `linear_solver`.
+
+- System attribute `metadata` is renamed `options`.
+
+- `units = 'unitless'` is removed.  Use `units = None` instead.
+
+- NewtonSolver attribute `line_search` is now `linesearch`.
+
```

Comment:
Add:

-  IndepVarComp: removed multi-variable specification via a list or tuple. Use add_output for each variable instead.
- Driver: removed method 'set_simul_deriv_color'. Use 'use_fixed_coloring' instead.
- Problem: 'vector_class' removed. Use 'distributed_vector_class' instead.
- pyOptSparseDriver: 'dynamic_simul_derivs' removed. Use 'declare_coloring' instead.